packages/libs/react-components/src/styles/stitches.config.ts

```diff
@@ -69,6 +75,7 @@ export const {
       ...colors,
       background: '$colors$neutral1',
       foreground: '$colors$neutral6',
+      borderColor: '#70797b33',
```

Comment:
should this be one of the neutral colors? My understanding is that all colors will have a token, but then we can give them an additional alias so it's easy to review them